portfolio/src/main/webapp/data.js

```diff
@@ -1,66 +1,24 @@
 // arrays store text that will be displayed on pages 
 window.moments = [
-    `I did math research on knots in high school, 
-    so my goal was to prove new patterns about these 
-    geometrical figures. So, I spent a lot of time 
-    drawing a lot of diagrams to test out new ideas.`,
+    `I did math research on knots in high school, so my goal was to prove new patterns about these geometrical figures. So, I spent a lot of time drawing a lot of diagrams to test out new ideas.`,
```

Comment:
Please revert these. Your javascript should never have more than 80 characters on a line.

Reply:
When added line breaks previously then these line breaks would show up in the rendered page. I didn't know how to change it so I kept the line long.

Reply:
```
const moments = [
  'some text that is'
  + 'going to be very long.',
  ....
];
```